homeassistant/components/influxdb/__init__.py

```diff
@@ -343,7 +343,12 @@ def get_influx_connection(conf, test_write=False, test_read=False):
         def write_v2(json):
             """Write data to V2 influx."""
             try:
-                write_api.write(bucket=bucket, record=json, write_precision=precision)
+                if precision is not None:
+                    write_api.write(
+                        bucket=bucket, record=json, write_precision=precision
+                    )
+                else:
+                    write_api.write(bucket=bucket, record=json)
```

Comment:
Store the parameters in a dict and do dict unpack in the function call instead. That way the function call will be the same.

Reply:
Can you check if this is what you mean? If so, I'll fix tests which expect `None` for `write_precision`.